database/migrations/2022_11_07_171403_add_config_descriptions.php

```diff
@@ -0,0 +1,116 @@
+		DB::table('configs')->where('key', 'location_decoding_timeout')->update(['description' => 'Timeout for the GPS decoding queries']);
+		DB::table('configs')->where('key', 'location_show')->update(['description' => 'Show location extracted from GPS coordinates']);
+		DB::table('configs')->where('key', 'location_show_public')->update(['description' => 'Anonymous users can acess the extracted location from GPS coordinates']);
+		DB::table('configs')->where('key', 'rss_enable')->update(['description' => 'Enable RSS feed']);
+		DB::table('configs')->where('key', 'rss_recent_days')->update(['description' => 'Display the last X days in the RSS feed']);
+		DB::table('configs')->where('key', 'rss_max_items')->update(['description' => 'Max number of itmes in the RSS feed']);
+		DB::table('configs')->where('key', 'prefer_available_xmp_metadata')->update(['description' => 'Use sidecard if provided instead of exif metadata']);
+		DB::table('configs')->where('key', 'editor_enabled')->update(['description' => 'Enable manual rotation of images']);
+		DB::table('configs')->where('key', 'lossless_optimization')->update(['description' => 'Apply compression on images']);
+		DB::table('configs')->where('key', 'swipe_tolerance_x')->update(['description' => 'Defines default horizontal swipe tolerance for mobile interactions']);
+		DB::table('configs')->where('key', 'swipe_tolerance_y')->update(['description' => 'Defines default vertical swipe tolerance for mobile interactions']);
+		DB::table('configs')->where('key', 'log_max_num_line')->update(['description' => 'Display the last X more recent lines in Logs']);
+		DB::table('configs')->where('key', 'unlock_password_photos_with_url_param')->update(['description' => 'Allow password to be passed as Url param to unlock albums']);
+		DB::table('configs')->where('key', 'nsfw_visible')->update(['description' => 'Make sensitive albums visible by default']);
+		DB::table('configs')->where('key', 'nsfw_blur')->update(['description' => 'Blur sensitive albums']);
+		DB::table('configs')->where('key', 'nsfw_warning')->update(['description' => 'Enable sensitive albums warning']);
+		DB::table('configs')->where('key', 'nsfw_warning_admin')->update(['description' => 'Enable sensitive albums warning when logged in']);
+		DB::table('configs')->where('key', 'nsfw_warning_text')->update(['description' => 'Text of the sensitive albums warning']);
+		DB::table('configs')->where('key', 'map_display_direction')->update(['description' => 'Display the direction of he picture on the map if available']);
+		DB::table('configs')->where('key', 'album_subtitle_type')->update(['description' => 'Defines the subtitle of album in albums view']);
+		DB::table('configs')->where('key', 'upload_processing_limit')->update(['description' => 'Maximum number of image processed in parallel']);
+		DB::table('configs')->where('key', 'public_photos_hidden')->update(['description' => 'Includes singular public pictures in Search results']);
```

Comment:
```suggestion
		DB::table('configs')->where('key', 'public_photos_hidden')->update(['description' => 'Keep singular public pictures hidden']);
```
It's not just search but also smart albums. I figured it was better to keep it short.

Reply:
Fixed: [6933807](https://github.com/LycheeOrg/Lychee/pull/1584/commits/693380795f25a5aeec84a6fb02524721e2def2b1)